server/partyManager.go

```diff
@@ -56,22 +59,42 @@ func (pm *PartyManager) Party(pid uuid.UUID) (*party.Party, error) {
 	// either way the user should behave the same way
 	p, found := pm.parties[pid]
 	if !found {
-		return nil, fmt.Errorf("could not find party %s", pid.String())
+		return nil, fmt.Errorf("could not find party %s", pid)
 	}
 
 	return p, nil
 }
 
 // Remove a party from the manager by uuid
-func (pm *PartyManager) Remove(pid uuid.UUID) error {
+func (pm *PartyManager) Remove(pid PartyUUID) error {
 	pm.mux.Lock()
 	defer pm.mux.Unlock()
 
 	if _, found := pm.parties[pid]; !found {
-		return fmt.Errorf("could not find party %s", pid.String())
+		return fmt.Errorf("could not find party %s", pid)
 	}
 
 	// NOTE: disbanding a party is the same as it not existing
 	delete(pm.parties, pid)
 	return nil
 }
+
+// generateUUID with 6 letters / numbers
+// panics if can't create a uuid
+func (pm PartyManager) generateUUID() PartyUUID {
+	letterBytes := "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+	// try to generate a uuid
+	for j := 0; j < 10; j++ {
+		b := make([]byte, 6)
```

Comment:
Number of characters in the id, I can move this out to a param, or we can make it const